src/controllers/tripsController.js

```diff
@@ -1,11 +1,14 @@
 import uuid from 'uuid/v4';
+import dotenv from 'dotenv';
 import db from '../models';
 import Response from '../utils/ResponseHandler';
 import TripsService from '../services/tripServices';
 import stringHelper from '../utils/stringHelper';
 import notifService from '../services/notificationService';
 import SendNotification from '../utils/sendNotification';
 
+dotenv.config();
+
```

Comment:
do we need this dotenv?

Reply:
I needed it to update the notification link, but now, it is no longer needed